pyproject.toml

```diff
@@ -327,6 +327,84 @@ sktime = [
 exclude = ["tests", "tests.*"]
 
 [tool.nbqa.exclude]
-black = "^docs/source/examples/"
-flake8 = "^docs/source/examples/"
-isort = "^docs/source/examples/"
+black = "^docs/source/examples/|^examples/"
+flake8 = "^docs/source/examples/|^examples/"
+isort = "^docs/source/examples/|^examples/"
```

Comment:
Since you removed `nbqa` now, does it have any effect?

Reply:
yes they're to be removed